src/js/actions/deviceActions.js

```diff
@@ -105,13 +105,18 @@ const filterProcessors = {
   $lt: val => Number(val) || val,
   $lte: val => Number(val) || val,
   $in: val => ('' + val).split(','),
-  $nin: val => ('' + val).split(',')
+  $nin: val => ('' + val).split(','),
+  $exists: () => 1,
+  $nexists: () => 0
+};
+const filterAliases = {
+  $nexists: () => '$exists',
```

Comment:
this might be enough:
```suggestion
  $nexists: '$exists'
```
then the access is just `filterAliases[filter.operator]`

Reply:
Done!